src/mavedb/lib/score_sets.py

```diff
@@ -93,10 +93,9 @@ def search_score_sets(db: Session, owner: Optional[User], search: ScoreSetsSearc
 
     if search.target_organism_names:
         query = query.filter(
-            ScoreSet.target_genes.any(
-                TargetGene.target_sequence.has(
-                    TargetSequence.reference.has(ReferenceGenome.organism_name.in_(search.target_organism_names))
-                )
+            ScoreSet.target_gene.target_sequence.has(
```

Comment:
why isn't this target_genes.any( as before?

Reply:
typo